operations/mimir-tests/test-read-write-deployment-mode-s3.jsonnet

```diff
@@ -0,0 +1,24 @@
+local mimir = import 'mimir/mimir.libsonnet';
+
+mimir {
+  _config+:: {
+    namespace: 'default',
+    external_url: 'http://test',
+    aws_region: 'eu-west-1',
+
+    storage_backend: 's3',
+    blocks_storage_bucket_name: 'blocks-bucket',
+    bucket_index_enabled: true,
```

Comment:
Nit: this is the default value, I'd skip it.

Reply:
Agree. Will do in a separate PR, cause affects other tests too.

Reply:
See: https://github.com/grafana/mimir/pull/3388